tests/test_transport.py

```diff
@@ -128,8 +139,8 @@ def test_transport_works(
     if use_pickle:
         client = pickle.loads(pickle.dumps(client))
 
-    sentry_sdk.Scope.get_global_scope().set_client(client)
-    request.addfinalizer(lambda: sentry_sdk.Scope.get_global_scope().set_client(None))
+    get_global_scope().set_client(client)
```

Comment:
I would like it more to always call `sentry_sdk.get_x()` to make it more explicit. But your call.